.gitlab/new-e2e_common/testing.yml

```diff
@@ -27,18 +27,18 @@
     TARGETS: ./tests/agent-platform/package-signing
     TEAM: agent-platform
     EXTRA_PARAMS: --osversion $E2E_BRANCH_OSVERS
-    
+
 .new-e2e_script_upgrade6:
   variables:
     TARGETS: ./tests/agent-platform/upgrade
     TEAM: agent-platform
     EXTRA_PARAMS: --osversion $E2E_OSVERS --platform $E2E_PLATFORM --arch $E2E_ARCH --flavor $FLAVOR
   parallel:
     matrix:
-      - START_MAJOR_VERSION: [5,6]
+      - START_MAJOR_VERSION: [5, 6]
         END_MAJOR_VERSION: [6]
   script:
-    - export DATADOG_AGENT_API_KEY=$(aws ssm get-parameter --region us-east-1 --name ci.agent-linux-install-script.datadog_api_key --with-decryption --query "Parameter.Value" --out text)
+    - export DATADOG_AGENT_API_KEY=$(source tools/ci/aws_ssm_get_wrapper.sh ci.agent-linux-install-script.datadog_api_key)
```

Comment:
This one seems different from the others, did you intentionally use `source` here?

Reply:
nice catch, it was a preliminary version, updating